webapp/player-project/src/streamers/index.ts

```diff
@@ -17,3 +17,12 @@ export const getShadowPlayer = (fileType) => {
 
   return player;
 };
+
+export const cleanUpStreamers = () => {
+  // Clean up any existing shadow-player elements
```

Comment:
suggestion: This comment is probably unnecessary, but if you want to keep it, here is a better wording.
```suggestion
  // Remove all shadow-player elements.
```

Reply:
Note that in this case, it’s much more useful to describe the _why_, rather than the _what_. Maybe it makes more sense to explain the why at the callsite of this function.